docs/docsource/reference/access_controls/bucket_policies/index.rst

```diff
@@ -0,0 +1,21 @@
+Bucket Policy
+=============
+
+Bucket policies afford you the opportunity to restrict or permit access to
+buckets based on policies owned by the account that owns a bucket. Account users
```

Comment:
While it's technically correct, I feel like this first sentence is a little confusing. I think something like "...based on policies set on a bucket by the bucket owner account". The idea of policies being owned by the account could be a bit misleading

Reply:
yeah... I was kind of worried about this. The issue (and correct me if my understanding is wrong) is that there aren't users so much as accounts  in Zenko, so the account/user split is nerve-wracking to say the least. 

If I don't hear any more on this, I'll change it to "user".

Reply:
Strategic vagueness to the rescue: 

"based on policies set by a bucket's owner."